Update ACBA bank + Add 4 new Armenian banks

MKo-xx opened the pull request:
1) Update ACBA bank tags.
2) Add 4 new Armenian banks: 
    a) Ameriabank
    b) Araratbank
    c) Ardshinbank
    d) Armbusinessbank

UKChris-osm commented:
Hi @MKo-xx 

Your PR is failing our tests because after each new `name` tag you have added, you need to add a comma at the end of each one (except the last one).

MKo-xx commented:
> Hi @MKo-xx
> 
> Your PR is failing our tests because after each new `name` tag you have added, you need to add a comma at the end of each one (except the last one).

Hi @UKChris-osm
Thanks for feedback. I just fixed the syntax issues and added few more banks info also.
